symphony/bdk/gen/rest.py

```diff
@@ -49,8 +49,12 @@ def __init__(self, configuration, pools_size=4, maxsize=None):
             maxsize = configuration.connection_pool_maxsize
 
         ssl_context = ssl.create_default_context(purpose=ssl.Purpose.CLIENT_AUTH, cafile=configuration.ssl_ca_cert)
```

Comment:
if `cafile` is None, it uses the system's default CA, see https://docs.python.org/3/library/ssl.html#ssl.create_default_context